serpentTools/objects/materials.py

```diff
@@ -103,7 +103,7 @@ def addData(self, variable, rawData):
 
     @messages.depreciated
     def getXY(self, xUnits, yUnits, timePoints=None, names=None):
-        """Depreciated. Use getValues instead"""
+        """Deprecated. Use getValues instead"""
```

Comment:
[depreciated](https://www.merriam-webster.com/dictionary/depreciated)

Reply:
are you sure? This is the word I've typically seen in software contexts. Sorry about the semantics. :stuck_out_tongue: https://techterms.com/definition/deprecated